src/components/v2/appDetails/k8Resource/k8resources.scss

```diff
@@ -15,14 +15,13 @@
  */
 
 .k8-resources-node-tree {
-    width: 270px;
-    min-width: 260px;
+    flex: 0 0 270px;
     min-height: 532px;
     overflow-y: auto;
 }
 
 .resource-node-wrapper {
-    min-height: calc(100vh - 168px);
+    min-height: calc(100vh - 120px);
 }
 
 .node-container-fluid {
```

Comment:
suggestion: let's clean this as well if possible

Reply:
this class is being used at multiple places